src/utils/traffic-mesh.js

```diff
@@ -92,6 +97,81 @@ const startForwardProxy = async ({ port, frameworkPort, functionsPort, publishDi
   }
 }
 
+const forwardMessagesToLog = ({ log, subprocess }) => {
+  let currentId = null
+  let spinner = null
+
+  const reset = () => {
+    currentId = null
+    spinner = null
+  }
+
+  rl.createInterface(subprocess.stderr)
```

Comment:
Should this use [`historySize: 0`](https://nodejs.org/api/readline.html#readline_readline_createinterface_options)?